pkg/local_workflows/code_workflow/native_workflow.go

```diff
@@ -95,12 +99,27 @@ func defaultAnalyzeFunction(target scan.Target, httpClientFunc func() *http.Clie
 		codeclient.WithLogger(logger),
 	)
 
-	result, _, err = codeScanner.UploadAndAnalyze(ctx, interactionId, target, files, changedFiles)
+	var bundleHash string
+	result, bundleHash, err = codeScanner.UploadAndAnalyze(ctx, interactionId, target, files, changedFiles)
+	if bundleHash == "" {
+		logger.Warn().Msg("Bundle hash is empty")
+		return result, snyk_errors.Error{
+			ID:             "Snyk-CLI-0002",
+			Title:          "No Supported Projects Detected",
+			Classification: "ACTIONABLE",
+			Level:          "warning",
+			Detail:         "We found 0 supported files\nPlease see our documentation for Snyk Code language and framework support\n",
+			Links: []string{
+				"https://docs.snyk.io/products/snyk-code/snyk-code-language-and-framework-support",
+			},
+		}
```

Comment:
**issue:** Stubbed out error catalog object, introduce error upstream then import and replace here.